demos/index.html

```diff
@@ -14,19 +13,63 @@
         <script type="text/javascript" src="../src/js/dataEntry.js"></script>
         <script type="text/javascript" src="../src/js/dataEntryPanel.js"></script>
         <script type="text/javascript" src="../src/js/chartAuthoring.js"></script>
+        <script type="text/javascript" src="../src/js/d3ViewComponent.js"></script>
+        <script type="text/javascript" src="../src/js/legend.js"></script>
+        <script type="text/javascript" src="../src/js/pie.js"></script>
+        <script type="text/javascript" src="../src/js/pieChart.js"></script>
+        <script type="text/javascript" src="../src/js/d3Utils.js"></script>
+
 
         <title>Chart Authoring Tool</title>
+        <style>
+            .floec-ca-pieChart-pie {
+                float: left;
+                width: 400px;
+            }
+            .floec-ca-pieChart-legend {
```

Comment:
See the comment above regarding class name conventions. Same comment applies to line 47.